config/test.js

```diff
@@ -0,0 +1,18 @@
+const test = {
+  network: {
+    name: 'Test',
+    rest: ['https://api.n06.testnet.vega.xyz', 'https://api.n07.testnet.vega.xyz'],
+    console: 'https://console.fairground.wtf',
+    explorer: 'https://explorer.fairground.wtf',
+    mockNodePort: 9090
```

Comment:
Can we move this under a property call test or export a secondary object from here with just test config? Mock server ports aren't really related to network.

Reply:
Agree, just toss it into the URL itself